spec/models/comment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```

Comment:
Do you not have a feature test that the comment is working and passing? It looks like you just have the rails generated spec test which are empty.